src/manager.js

```diff
@@ -819,6 +824,7 @@ AdManager.prototype.refreshSlots = function (slotsToLoad) {
  * @returns undefined
 */
 AdManager.prototype.prebidRefresh = function (slots) {
+  this.pbjs = window.pbjs;
```

Comment:
this mirrors:

https://github.com/theonion/bulbs-public-ads-manager/blob/15762303cc7c56c61ab9bcc23224d6717820613c/src/manager.js#L76